common/djangoapps/track/backends/logger.py

```diff
@@ -33,7 +34,14 @@ def __init__(self, name, **kwargs):
         self.event_logger = logging.getLogger(name)
 
     def send(self, event):
-        event_str = json.dumps(event, cls=DateTimeJSONEncoder)
+        try:
+            event_str = json.dumps(event, cls=DateTimeJSONEncoder)
+        except UnicodeDecodeError:
+            application_log.exception(
+                "UnicodeDecodeError Event_type: %r, Event_source: %r, Page: %r, Referer: %r",
+                event.get('event_type', 'None'), event.get('event_source', 'None'),
+                event.get('page', 'None'), event.get('referer', 'None')
```

Comment:
```get()``` always return ```None``` as default value if key is not present in the dict, why you're passing it manually?